support/rally.js

```diff
@@ -176,6 +211,12 @@ export default class Rally {
       return;
     }
 
+    // When paused, not send data.
```

Comment:
Note to self - move this from `rally.js` into `Core.js`, so we do not need to trust the (potentially misbehaving) study extension.

Reply:
Hm. Actually I'd like to defer this one, I think we don't have the information we need. I'll send you a message about this as it is a little involved.